matchbox/resource_profile.go

```diff
@@ -193,5 +236,9 @@ func containerLinuxConfig(d *schema.ResourceData) (filename, config string) {
 		return fmt.Sprintf("%s.ign", name), content.(string)
 	}
 
+	if content, ok := d.GetOk("generic"); ok {
+		return fmt.Sprintf("%s.generic", name), content.(string)
```

Comment:
There is no requirement that generic templates end in a specific suffix, this should be removed.